### PR TITLE
docs(azure): how to use Azure Cloud Shell

### PR DESCRIPTION
**Pull request by afiune**
Signed-off-by: Salim Afiune Maya <afiune@lacework.net>

### CODE REVIEW

File: azure/AZURE_CLOUD_SHELL.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Azure Cloud Shell
+The Azure Cloud Shell is an embedded terminal/command-line interface that can be used within the Azure
+Portal. Once you access the Cloud Shell it automatically authenticates you with Azure AD, additionally,
+it has the Azure CLI and some other popular open source tools, like Terraform.
```
Comment (scottford-lw) [inline azure/AZURE_CLOUD_SHELL.md:4]: I would update this to say...

```
Cloud Shell automatically authenticates you with Azure AD and comes with tools like the Azure CLI and Terraform pre-installed to manage and automate your Azure environment.
```

File: azure/AZURE_CLOUD_SHELL.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+These instructions will show you how to get up-and-running with Lacework, Terraform and the Azure Cloud Shell.
+
+## Open Azure Cloud Shell within Azure Portal
+To open the Azure Cloud Shell, you simply click on the Cloud Shell icon in the header bar of the Azure Portal,
+and it will open the Cloud Shell in a pane at the bottom of the browser. By default it configures a PowerShell
+prompt but you could switch to a Bash prompt.
```
Comment (scottford-lw) [inline azure/AZURE_CLOUD_SHELL.md:11]:
```suggestion
Azure Shell defaults to PowerShell, but you can also switch to a Bash prompt if you prefer.
```

File: azure/AZURE_CLOUD_SHELL.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+PS /home/salim> mkdir -p $env:HOME/bin
+PS /home/salim> $env:PATH = "$env:HOME/bin:$env:PATH"
+```
+
+__NOTE: If you are using a Bash prompt you will have to substitute the environment variable `$env:HOME` and `$env:PATH`
+for `$HOME` and `$PATH` respectively.__
+
+## Install the Lacework CLI
+The Lacework Command Line Interface is a tool that helps you manage the Lacework cloud security platform. You can use
+it to manage compliance reports, external integrations, vulnerability scans, and other operations.
```
Comment (scottford-lw) [inline azure/AZURE_CLOUD_SHELL.md:31]:
```suggestion
The Lacework Command Line Interface is a tool that helps you manage the Lacework cloud security platform. You can use it to manage compliance reports, external integrations, vulnerability scans, and other operations. Once configured, the Lacework's Terraform provider can leverage the CLI's configuration including API keys for accounts you want to configure.
```

File: azure/AZURE_CLOUD_SHELL.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+module "az_activity_log" {
+  source = "git::https://github.com/lacework/terraform-provisioning.git//azure/modules/activity_log?ref=master"
+
+  use_existing_ad_application = true
+  application_id              = module.az_config.application_id
+  application_password        = module.az_config.application_password
+  service_principal_id        = module.az_config.service_principal_id
+}
+```
```
Comment (scottford-lw) [inline azure/AZURE_CLOUD_SHELL.md:107]: @afiune should we mention something about version pinning here? `ref=tags/v0.2.1` We might want to have link to this...

https://github.com/lacework/terraform-provisioning/releases

**Review by scottford-lw — APPROVED** [review]
Amazing work @afiune

File: azure/AZURE_CLOUD_SHELL.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
+# Azure Cloud Shell
+The Azure Cloud Shell is an embedded terminal/command-line interface that can be used within the Azure
+Portal.  Cloud Shell automatically authenticates you with Azure AD and comes with tools like the Azure
+CLI and Terraform pre-installed to manage and automate your Azure environment.
+
+These instructions will show you how to get up-and-running with Lacework, Terraform and the Azure Cloud Shell.
+The only requirement we need is that your [Azure User](https://cloud.google.com/iam/docs/service-accounts) has
+the following permissions:
+- **Global Administrator** privileges in Active Directory
+- **Owner Role** at the Subscription level
```
[the remark this answers is not in the document]
Comment (afiune) [reply azure/AZURE_CLOUD_SHELL.md:10]: Yes, they both are! 💯

**Pull request merged**